docs/reference/modules/http.asciidoc

```diff
@@ -72,6 +72,10 @@ be cached for. Defaults to `1728000` (20 days)
 header should be returned. Note: This header is only returned, when the setting is
 set to `true`. Defaults to `false`
 
+|`http.emit_stack_trace` |Enables or disables the output of stack traces in response
+output. Note: When set to `false` this setting overrides the `error_trace` request parameter
+and no stack trace will be output; only a simple message will be displayed. Defaults to `true`
```

Comment:
If we do that then we need to make sure the tests will always override this to true.